src/GeolocatorService.Abstractions/GeolocatorService.Abstractions.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFrameworks>netstandard2.0</TargetFrameworks>
+		<!-- Ensures the .xr.xml files are generated in a proper layout folder -->
+		<GenerateLibraryLayout>true</GenerateLibraryLayout>
+		<RootNamespace>GeolocatorService</RootNamespace>
+		<Authors>nventive</Authors>
+		<Company>nventive</Company>
+		<AssemblyName>GeolocatorService.Abstractions</AssemblyName>
+		<PackageId>GeolocatorService.Abstractions</PackageId>
+		<Description>GeolocatorService</Description>
+		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageTags>geolocation;gps;abstractions;mvvm;xamarin;uwp;ios;android</PackageTags>
+		<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+		<PackageProjectUrl>https://github.com/nventive/GeolocatorService</PackageProjectUrl>
+
+		<!--Needed for Source Link support -->
+		<PublishRepositoryUrl>true</PublishRepositoryUrl>
+		<EmbedUntrackedSources>true</EmbedUntrackedSources>
+		<IncludeSymbols>true</IncludeSymbols>
+		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
+	</PropertyGroup>
+
+	
+	<ItemGroup>
```

Comment:
You don't need those xaml-related stuff since an Abstractions package will never contain any xaml.